sdk/test/ProtocolTests/AWSSDK.ProtocolTests.Netstandard.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+     <RunAnalyzersDuringBuild Condition="'$(RunAnalyzersDuringBuild)'==''">true</RunAnalyzersDuringBuild>
+      <TargetFrameworks>netstandard2.0;netcoreapp3.1;net8.0</TargetFrameworks>
+      <DefineConstants>$(DefineConstants);TRACE;BCL;BCL45;ASYNC_AWAIT;DEBUG</DefineConstants>
```

Comment:
Is including `BCL` and `BCL45` in this .NET Standard project correct? I don't see these constants defined in other test projects (e.g. https://github.com/aws/aws-sdk-net/blob/main/sdk/test/NetStandard/UnitTests/AWSSDK.UnitTests.Custom.NetStandard.csproj#L10)

Reply:
Corrected it to the sample in the other test project.